vaadin-popover-flow-parent/vaadin-popover-flow/src/main/java/com/vaadin/flow/component/popover/Popover.java

```diff
@@ -60,6 +68,48 @@ public class Popover extends Component implements HasComponents {
      */
     public Popover() {
         getElement().getNode().addAttachListener(this::attachComponentRenderer);
+
+        setTrigger(Set.of(PopoverTrigger.CLICK));
+    }
+
+    /**
+     * Sets the popover trigger mode describing how the overlay is opened or
+     * closed on user interaction with the target.
+     *
+     * When passing null or an empty set, the popover can only be opened and
+     * closed programmatically.
+     *
+     * @param trigger
+     *            the set of trigger values to set
+     */
+    public void setTrigger(Set<PopoverTrigger> trigger) {
+        JsonArray triggerAsJson = Json.createArray();
+
+        if (!(trigger == null || trigger.isEmpty())) {
+            triggerAsJson = JsonSerializer.toJson(trigger.stream()
+                    .map(PopoverTrigger::toString).collect(Collectors.toSet()));
+        }
+
+        getElement().setPropertyJson("trigger", triggerAsJson);
+    }
+
+    /**
+     * Gets the popover trigger mode describing how the overlay is opened or
+     * closed on user interaction with the target.
```

Comment:
Let's mention what the default configuration for the trigger is.

Reply:
Removed this block so it's no longer relevant.